projects/email-editor/src/lib/source.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-email-editor",
-  "version": "15.0.0",
+  "version": "15.1.0",
   "peerDependencies": {
     "@angular/common": "^15.0.0",
```

Comment:
I assume the answer is yes, but is this version compatible with clients on angular 18?

Reply:
yes, it works with angular 18